### PR TITLE
fix: use mini-migrators for everything

### PR DESCRIPTION
**Pull request by beckermr**
#### Description:

This PR adds a set of default mini-migrators for every thing we can.

#### Checklist:

- [x] Pydantic model updated or no update needed

### CODE REVIEW

**Comment by beckermr** [comment]
cc @h-vetinari this might help with not missing additions in the future

File: conda_forge_tick/make_migrators.py [hunk 1]
```diff
@@ -97,6 +98,29 @@
 MAX_PR_LIMIT = 50
 MAX_SOLVER_ATTEMPTS = 50
 CHECK_SOLVABLE_TIMEOUT = 90  # 90 days
+DEFAULT_MINI_MIGRATORS = [
```
Comment (h-vetinari) [inline conda_forge_tick/make_migrators.py:101]: Thanks for this!

What was the reason to not add `StdlibMigrator` to the default set? It's in use for both types of migrators (after #2372 & #2459)

File: conda_forge_tick/make_migrators.py [hunk 2]
```diff
@@ -777,22 +796,13 @@ def initialize_migrators(
...
-                DependencyUpdateMigrator(python_nodes),
-                PyPIOrgMigrator(),
-                StdlibMigrator(),
-            ],
+            piggy_back_migrations=_make_mini_migrators_with_defaults(
+                extra_mini_migrators=[
+                    PipWheelMigrator(),
+                    DependencyUpdateMigrator(python_nodes),
+                    StdlibMigrator(),
+                ],
```
Comment (h-vetinari) [inline conda_forge_tick/make_migrators.py:804]: I see it gets readded here anyway, but that just sounds to me like it should be in the default set? 🤔

Comment (beckermr) [reply conda_forge_tick/make_migrators.py:804]: I tried to separate formatting versus recipe behavior changes, with the goal of future additions to the migrators preferring one over the other depending on context.

Comment (h-vetinari) [reply conda_forge_tick/make_migrators.py:804]: OK, fair enough. 🤷

**Pull request merged**